src/polar/Radar.tsx

```diff
@@ -84,6 +86,29 @@ type RadarComposedData = {
   isRange: boolean;
 };
 
+function getLegendItemColor(stroke: string | undefined, fill: string): string {
+  return stroke && stroke !== 'none' ? stroke : fill;
```

Comment:
where was this done previously? getMainColorOfGraphicItem?

Reply:
Correct